File: Cooper.API/Response/Challenge/DeleteChallengeResponse.cs
using System.Net;
using Cooper.API.Common;

namespace Cooper.API.Response.Challenge
{
    public class DeleteChallengeResponse:ResponseBase
    {

        public DeleteChallengeResponse(HttpStatusCode httpStatusCode, string description):base(httpStatusCode, description){
        }

       public Common.Challenge Challenge{get; set;}
    }
}

File: Cooper.Mobile/src/app/core/interface/http/challenge/response/DeleteChallenge.response.ts
import { ResponseBase } from "../../common"

export interface DeleteChallengeResponse extends ResponseBase {
  challenge: {
    name: string,
    description: string,
    challengeId:string,
    title: string,
    startDate: string,
    endDate:string,
    price: number
  }
}
